src/commands/orders.py

```diff
@@ -0,0 +1,197 @@
+  else:
+    raise Exception('Format "%s" is not supported. Supported formats are: pretty, csv' % (format))  
+
+
+def to_order_dto(order):
+  return {
+    "owner_address": order['owner']['id'],
+    "order_id": int(order['orderId']),
+    "fromBatchId": int(order['fromBatchId']),
+    "untilBatchId": int(order['untilBatchId']),
+    "sellToken": to_token(order['sellToken']),
+    "buyToken": to_token(order['buyToken']),
+    "priceNumerator": Decimal(order['priceNumerator']),
+    "priceDenominator": Decimal(order['priceDenominator']),
+    "maxSellAmount": Decimal(order['maxSellAmount']),
+    "soldVolume": Decimal(order['soldVolume']),
+    "boughtVolume": Decimal(order['boughtVolume']),
+    "createDate": parse_date_from_epoch(order['createEpoch']),
+    "cancelDate": parse_date_from_epoch(order['cancelEpoch']),
+    "deleteDate": parse_date_from_epoch(order['deleteEpoch']),
+    "txHash": order['txHash']
+  }
+
+def print_orders_pretty(orders):
+  click.echo(click.style(SEPARATOR, fg=COLOR_SEPARATOR))
+
+  for order in orders:
+    cancelDate, deleteDate = order['cancelDate'], order['deleteDate']
+    priceNumerator, priceDenominator = order['priceNumerator'], order['priceDenominator']
+    sellToken, soldVolume = order['sellToken'], order['soldVolume']
+    buyToken, boughtVolume = order['buyToken'], order['boughtVolume']
+    sellTokenDecimals, sellTokenLabel = sellToken['decimals'], format_token_short(sellToken)
+    buyTokenDecimals, buyTokenLabel = buyToken['decimals'], format_token_short(buyToken)
+
+    labelColor = COLOR_LABEL
+    if cancelDate is None:
+      cancelDateText = ''
+    else:
+      labelColor = COLOR_LABEL_DELETED
+      cancelDateText = click.style('  Cancel date', fg=labelColor) + ': ' + click.style(format_date_time(cancelDate), bg=COLOR_LABEL_DELETED) + '\n'
+
+    if deleteDate is None:
+      deleteDateText = ''
+    else:
+      labelColor = COLOR_LABEL_DELETED
+      deleteDateText = click.style('  Deleted date', fg=labelColor) + ': ' + click.style(format_date_time(deleteDate), bg=COLOR_LABEL_DELETED) + '\n'
+
+    tradePriceText = ''
+    if soldVolume > 0:
+      tradePriceText = (
+        click.style(f'  Avg. Traded Price {sellTokenLabel}/{buyTokenLabel}', fg=labelColor) + ': ' + 
+        format_price(
+          calculate_price(
+            numerator=boughtVolume,
+            denominator=soldVolume,
+            decimals_numerator=buyTokenDecimals,
+            decimals_denominator=sellTokenDecimals
+          ), 
+          currency=buyTokenLabel
+        ) +
+        '\n' +
+        click.style(f'  Avg. Traded Price {buyTokenLabel}/{sellTokenLabel}', fg=labelColor) + ': ' + 
+        format_price(
+          calculate_price(
+            numerator=soldVolume,
+            denominator=boughtVolume,
+            decimals_numerator=sellTokenDecimals,
+            decimals_denominator=buyTokenDecimals
+          ),
+          currency=sellTokenLabel
+        ) +
+        '\n'
+      )
```

Comment:
Nit: I would put the prices into their own variables and simplify the string building to something like:

```python
price = format_price(
          calculate_price(
            numerator=boughtVolume,
            denominator=soldVolume,
            decimals_numerator=buyTokenDecimals,
            decimals_denominator=sellTokenDecimals
          ), 
          currency=buyTokenLabel
        )
priceStr1 = f'{click.style(f'  Avg. Traded Price {sellTokenLabel}/{buyTokenLabel}', fg=labelColor)}: {price1}'

...

tradePriceText = f'{priceStr1}\n{priceStr2}\n'
```

Reply:
https://github.com/gnosis/dex-cli/issues/39